src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java

```diff
@@ -30,20 +45,47 @@ public boolean hasTarget() {
         return limelight.getEntry("tv").getDouble(0) == 1;
     }
 
-    // x-axis
+    /**
+     * Returns the yaw from the limelight to the hub.
+     *
+     * @return The yaw (horizontal rotation) in degrees.
+     */
     public double getYaw() {
         return limelight.getEntry("tx").getDouble(0);
     }
 
-    // returns in inches
+    /**
+     * Returns the distance from the limelight to the hub.
+     *
+     * @return THe distance in inches.
+     */
     public double getDistance() {
-        double distance = HEIGHT_TO_RIM / Math.tan(LIMELIGHT_ANGLE_OFFSET + getPitch());
-        return distance;
+        return HEIGHT_TO_RIM / Math.tan(Math.toRadians(LIMELIGHT_ANGLE_OFFSET + getPitch()));
     }
 
-    // y-axis
+    /**
+     * Returns the pitch from the robot to the hub.
+     *
+     * @return The pitch (vertical rotation) in degrees.
+     */
     public double getPitch() {
-        return limelight.getEntry("ty").getDouble(0);
+        return limelight.getEntry("ty").getDouble(DEFAULT_PITCH);
+    }
+
+    /**
+     * Returns the estimated robot pose as a {@link RigidTransform2}.
+     *
+     * The translation (inches) is relative to the hub, and the rotation has the same reference as
```

Comment:
In the future, we should make sure that the poses returned from this method and from `DriveBaseSubsystem` are in the same frame of reference so we can easily combine both of them.

Reply:
I added `getEstimatedPoseRelativeToStart`, which converts the estimated pose to relative to the robot starting pose (using constants in `ShooterVisionConstants`).